Source/LoadByOS/LinuxConfigApp/LinuxInterop.cs

```diff
@@ -24,12 +25,17 @@ internal class LinuxInterop : IInteropFunctions
 		public void SetFolderIcon(string image, string directory) => throw new PlatformNotSupportedException();
 		public void DeleteFolderIcon(string directory) => throw new PlatformNotSupportedException();
 
+		public string ReleaseIdentifier => LibationScaffolding.ReleaseIdentifier.ToString() + (File.Exists("/bin/yum") ? "_RPM" : "");
```

Comment:
I like the adaptation here. It seems like it would be easy though to use `LibationScaffolding.ReleaseIdentifier` in code when you mean interop.ReleaseIdentifier

Reply:
Agreed. Changed it to `ReleaseIdString`.